daras_ai_v2/billing.py

```diff
@@ -414,57 +382,66 @@ def render_paypal_addon_buttons():
 
 
 def render_stripe_addon_buttons(user: AppUser):
+    if not (user.subscription and user.subscription.payment_provider):
+        save_pm = gui.checkbox(
+            "Save payment method for future purchases & auto-recharge", value=True
+        )
+    else:
+        save_pm = True
```

Comment:
isn't there a checkbox in the next step on stripe's page anyways?

Reply:
Stripe shows different payment options depending on this checkbox!

Reply:
The one on stripe is also easy to miss and unchecked by default